README.md

```diff
@@ -22,7 +22,12 @@ Copy `infrastructure/lambda/samconfig.toml.example` to `infrastructure/lambda/sa
 
 Deploy to the dev environment with:
 
-`gds aws di-ipv-cri-dev -- ./deploy.sh`
+`gds aws di-ipv-cri-dev -- ./deploy.sh your-stack-name`
+
+Supply a required stack name in place of `your-stack-name` above, the `CommonStackName` and `SecretPrefix`
```

Comment:
Update to indicate the common stack name & secret prefix are optional parameters